packages/react/src/components/Popup/Popup.tsx

```diff
@@ -122,62 +123,25 @@ function useRestoreFocus(props: IPopupProps, root: React.RefObject<HTMLDivElemen
 }
 
 function useHideSiblingNodes(props: IPopupProps, root: React.RefObject<HTMLDivElement | undefined>) {
-  const isModalOrPanel = props['aria-modal'];
   // eslint-disable-next-line deprecation/deprecation
-  const enableAriaHiddenSiblings = props.enableAriaHiddenSiblings;
+  const shouldHideSiblings = props['aria-modal'] && props.enableAriaHiddenSiblings;
 
   React.useEffect(() => {
-    const targetDocument = getDocument();
-    if (!(enableAriaHiddenSiblings && isModalOrPanel && targetDocument && root?.current)) {
+    if (!(shouldHideSiblings && root.current)) {
       return;
     }
 
-    const popupPortalNode = root.current.parentElement?.parentElement;
```

Comment:
@TristanWatanabe or @smhigley do you know if there's any of this logic that needs to be ported over to `utilities/src/modalize.ts`? In particular I wasn't sure about the logic to exclude template/script/style from hiding (what even happens if you hide those...?) but might have missed other things too.

Reply:
The template/script/style check is kinda nice because it avoids some unnecessary DOM mutation, though it ultimately won't affect the accessibility of the page. We should probably not move over the `!child.hasAttribute('aria-hidden')` check, since it looks like that would exclude elements with `aria-hidden="false"` (missed that earlier 😅)

Reply:
Yeah I guess reducing the DOM mutation doesn't hurt, especially if for some reason it would cause those tags to be re-processed (I don't know if this would happen or how to check). I'll add the template/script/style check back.